uvicorn/config.py

```diff
@@ -266,9 +272,10 @@ def configure_logging(self):
             logging.getLogger("uvicorn.access").handlers = []
             logging.getLogger("uvicorn.access").propagate = False
 
-    def load(self):
+    def load(self) -> None:
         assert not self.loaded
 
+        self.ssl: Optional[ssl.SSLContext]
         if self.is_ssl:
```

Comment:
Consider `if self.is_ssl and self.ssl_certfile:` instead, which will help ensure that the `self.ssl_certfile` required positional argument is present.